### PR TITLE
Create 2019-05-15-vvv-3-0-0.md

### PR DESCRIPTION
**Pull request by tomjn**
VVV 3 release post

### CODE REVIEW

**Review by ghost — APPROVED** [review]
looks good to me.

File: _posts/2019-05-15-vvv-3-0-0.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+```
+general:
+  db_share_type: false
+```
+
+We continued optimising provision in this release, and making VVV friendlier.
+
+### Vagrant Guest Additions Plugin Disabled
+
+Our new box has recent guest additions, so this shouldn't be needed. However, some users like this plugin, and when reprovisioning encountered problems. We've disabled the plugin just for VVV to avoid these issues.
```
Comment (Mte90) [inline _posts/2019-05-15-vvv-3-0-0.md:59]: I think that add the command to remove it can be useful

**Review by Mte90 — CHANGES_REQUESTED** [review]

File: _posts/2019-05-15-vvv-3-0-0.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+ - The Git PPA is now only added if it's missing
+ - Various composer and git commands now run as `root` ( VirtualBox determines the ownership of those folders, so they will always be owned by the `vagrant` user )
+ - The branch and repo URL are mentioned when cloning dashboards, sites, and utilities
+ - the VVV splash no longer shows on a `halt` or `suspend`
+ - when provisioning on Ubuntu 14 it aborts and gives instructions on updating
+ - added the VVV PPA package mirror
+ - nginx now gets reloaded instead of restarted
+ - VVV now copies over a minimal MySQL config for overrides
+ - Added `git-svn` support
+
```
Comment (Mte90) [inline _posts/2019-05-15-vvv-3-0-0.md:107]: it is missing the part of vvv.dev/vvv domains

**Review by Mte90 — CHANGES_REQUESTED** [review]

**Comment by Mte90** [comment]
I don't know if we want to mention also the new pages about tideways/contribute and also the updates to the template.

**Comment by Mte90** [comment]
are missing the 2 lasts changes: log output and missing tables for sql and I think that we can release finally

File: _posts/2019-05-15-vvv-3-0-0.md (hunk 1 above)
Comment (tomjn) [reply _posts/2019-05-15-vvv-3-0-0.md:59]: it should be ok, we tell the plugin not to run if it's present anyway, they might be able to fix it at their end too in the future

**Review by Mte90 — APPROVED** [review]

**Comment by Mte90** [comment]
We want to mention the 2 new pages about contribute and tideways?

**Comment by tomjn** [comment]
@Mte90 not in the release post, docs are on a continuous release cycle, but perhaps we can feature them in the dashboard?

**Comment by Mte90** [comment]
good idea

**Pull request merged**